8.1/Dockerfile.fedora

```diff
@@ -0,0 +1,80 @@
+FROM quay.io/fedora/s2i-base:36
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV PHP_VERSION=8.1 \
+    PHP_SHORT_VER=81 \
+    PATH=$PATH:/usr/bin
+
+ENV SUMMARY="Platform for building and running PHP $PHP_VERSION applications" \
+    DESCRIPTION="PHP $PHP_VERSION available as container is a base platform for \
+building and running various PHP $PHP_VERSION applications and frameworks. \
+PHP is an HTML-embedded scripting language. PHP attempts to make it easy for developers \
+to write dynamically generated web pages. PHP also offers built-in database integration \
+for several commercial and non-commercial database management systems, so writing \
+a database-enabled webpage with PHP is fairly simple. The most common use of PHP coding \
+is probably as a replacement for CGI scripts."
+
+ENV NAME=php \
+    VERSION=0 \
+    RELEASE=1 \
+    ARCH=x86_64
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Apache 2.4 with PHP $PHP_VERSION" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,php" \
+      name="fedora/$NAME-$PHP_SHORT_VER" \
+      com.redhat.component="$NAME" \
+      version="$VERSION" \
+      usage="s2i build https://github.com/sclorg/s2i-php-container.git --context-dir=/$PHP_VERSION/test/test-app quay.io/fedora/$NAME-$PHP_SHORT_VER sample-server" \
```

Comment:
quay.io/fedora/php-81 is already created. Waiting on robot account. https://quay.io/repository/fedora/php-81?tab=settings